cnd/src/http_api.rs

```diff
@@ -375,9 +352,9 @@ pub enum HttpAssetParams {
     Erc20(Erc20AssetParams),
 }
 
-#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
+#[derive(Clone, Copy, Debug, Serialize, Deserialize, PartialEq)]
 pub struct BitcoinAssetParams {
-    quantity: Http<asset::Bitcoin>,
```

Comment:
❓ Why the move away from the custom deserialisation?

Reply:
This was likely a mistake by me, have discussed with @thomaseizinger.  Am planning on removing this from the PR.

Reply:
So the upside of this is: left the custom serder in `bitcoin` module (because it is used by `strum_discriminants`/`HtlcState`), but did not use it in the http_api, instead reverted to using custom serde and wrapping in `Http` type. The `bitcoin` module serde stuff should be removed when we remove the `strum_discriminants` usage to create `HtlpState` (this relies on all types being serde).

Tracking issue: https://github.com/comit-network/comit-rs/issues/2636